test.c

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
```

Comment:
Is it necessary to include this file? If a C file is all you need, there are plenty in the repo itself. You could alternatively create one in the Dockerfile (`echo "int main() { return 0; }" > test.c`)